scrapers/KinkBomb.yml

```
@@ -0,0 +1,48 @@
+name: "KinkBomb"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - kinkbomb.com/p
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[@class="content-section"]
+      $metainfo: //div[@class="metainfo"]
+      $image: //div[@class="clip-preview-inner"]
+    scene:
+      Title: $details//h1/text()
+      Image:
+        selector: $image/img/@src|$image/video/@poster
+        postProcess:
+          - replace:
+              - regex: //
+                with: https://
+
+      Details:
+        selector: //div[@class="description-inner"]/p
+        concat: "\n\n"
+      Date:
+        selector: $metainfo/small[2]/text()
```

using a non indexed selector in cases like this seems safer
```suggestion
        selector: $metainfo/small[contains(span,"Posted")]/text()
```